KsApi/mutations/inputs/UpdateBackingInput.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+public struct UpdateBackingInput: GraphMutationInput, Encodable {
+  let amount: String?
+  let applePay: ApplePay?
+  let id: String
+  let locationId: String?
+  let paymentSourceId: String?
+  let rewardId: String?
+
+  public struct ApplePay: Encodable {
```

Comment:
Since this will be shared in `CreateBackingInput`, can we take it out of the `UpdateBackingInput` scope?

Reply:
Sure thing 👍